sklearn/metrics/_dist_metrics.pyx.tp

```diff
@@ -982,7 +982,7 @@ cdef class SEuclideanDistance{{name_suffix}}(DistanceMetric{{name_suffix}}):
     .. math::
        D(x, y) = \sqrt{ \sum_i \frac{ (x_i - y_i) ^ 2}{V_i} }
     """
-    def __init__(self, V):
+    def __init__(self, V, **kwargs):
```

Comment:
We do not commonly use this pattern to ignore `kwargs`, but I am okay with this, given that `DistanceMetric` is not "too public facing".

Reply:
This is fairly simple, but if you have a better suggestion, feel free to suggest it.

Reply:
If the consensus goes to accept ignored kwargs (which I personally don't like), we should at least make it explicit that those kwargs are ignored on purpose by renaming `**kwargs` to `**ignored_extra_kwargs`.

Reply:
6a0e229f4de73fc8b42ae63dad4973ac04c18459 revert this, and instead pops `Y_norm_squared` at an appropriate place.